### PR TITLE
first pass addition of serializeBodyFormat: smallcaps

### PR DESCRIPTION
**Pull request by warner**
refs #6822

### CODE REVIEW

File: packages/deploy-script-support/src/code-gen.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const defangAndTrim = code => {
   return trimmed;
 };
 
-const { serialize } = makeMarshal();
+const { serialize } = makeMarshal(undefined, undefined, { serializeBodyFormat: 'smallcaps' });
```
Comment (dckc) [inline packages/deploy-script-support/src/code-gen.js]: This one should not use smallcaps.

**Comment by warner** [comment]
tests are still failing in:
* quick:
  * governance: chip (done)
  * others after governance? (nope)
* vats: warner
* lint: chip (done)
* solo (appears to be ok)
* inter-protocol: chip (done)
* quick2:
  * wallet/api: chip (done)
  * others after wallet/api? (nope)

**Comment by warner** [comment]
I got packages/vats working with that last commit

**Review by erights — CHANGES_REQUESTED** [review]
Please switch to the approach illustrated at https://github.com/endojs/endo/blob/master/packages/marshal/test/test-marshal-testing.js unless there's a reason not to. It would do much less violence to the existing tests.

**Comment by warner** [comment]
abandoned in favor of #7585, which does much less violence to the existing tests

**Pull request closed without merging**